hardware_interface/src/mock_components/generic_system.cpp

```diff
@@ -57,16 +57,29 @@ CallbackReturn GenericSystem::on_init(const hardware_interface::HardwareInfo & i
     }
   };
 
-  // check if to create fake command interface for sensor
-  auto it = info_.hardware_parameters.find("fake_sensor_commands");
```

Comment:
@bmagyar should be deprecate this first?

Reply:
below it is still processed, the deprecation is in place IMO.

Can you confirm that is the intent @kvkpraneeth ?

Reply:
Yes, that was the intent